test/recipes/30-test_evp_data/evpmac.txt

```diff
@@ -159,6 +159,7 @@ Ctrl = digestsize:13
 Key = 000102030405060708090A0B0C0D0E0F
 Input = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F202122232425262728292A2B2C2D2E2F303132333435363738393A3B3C3D3E
 Output = 5150d1772f50834a503e069a973fbd7c
```

Comment:
Why is this showing output if it is meant to be catching an error?

Reply:
It doesn't, so the Input and Output lines are actually unnecessary...  unless removing them makes the parser fail, which I hadn't verified.  Coming up.

Reply:
(I have some rebase work to do anyway)

Reply:
Fixed.  The key must remain, though...

Reply:
Showing Output seems wrong - Input I can understand - but if the code doesn't make it to that part then I would expect to see zero length Output if the field has to be present.